### PR TITLE
security: add timeline, contact, and priority info

### PR DESCRIPTION
**Pull request by eslerm**
Late in the Mantic cycle Security approved three MIRs ([LP#2030482](https://bugs.launchpad.net/ubuntu/+source/s390-tools/+bug/2030482) and two in [LP#2038942](https://bugs.launchpad.net/ubuntu/+source/qrtr/+bug/2038942)). From this Security learned that we need to improve our documentation and set expectations.

@setharnold does Beta Freeze sound appropriate to you? That is tight, but feels workable.

@frank-heimes @xnox

### CODE REVIEW

**Comment by xnox** [comment]
I burned a lot of karma and brownie points because of that; and will still be paying for landing those........

I did not set a precedent.

**Comment by eslerm** [comment]
We were already discussing setting a timeline before the x13s packages.

> This brief security MIR is an exception. - [s390-tools](https://bugs.launchpad.net/ubuntu/+source/s390-tools/+bug/2030482/comments/6)

The director clause was added for exceptional cases, like hardware enablement.

The intent is to make things smoother, not to add blocks.

**Comment by setharnold** [comment]
Historically the security team has not been able to perform reviews with low enough latency and high enough throughput to deliver the necessary reviews early enough in the cycle -- we'd often be finishing them in the week before release, even if the consequences of the MIR succeeding or failing might require unrealistically large changes in short notice to implement or roll-back the changes.

We've wanted to have a submission deadline for years and it's perhaps fortuitous timing that we've got much more capacity at roughly the same as we've got some excellent examples of truly last-minute requests. :)

In an ideal world this wouldn't actually be much of a change -- hopefully everyone has the clear bulk of their work ready by feature freeze, and hopefully the MIR team has enough capacity to assign further security reviews as necessary, etc.

So, no, please don't take this personally -- but yes, the timing of those requests is a decent indicator of why we'd like the change.

Thanks

**Review by alexmurray — APPROVED** [review]
LGTM.

**Review by slyon — APPROVED** [review]
LGTM, except for the spelling fixes needed.

**Review by cpaelzer — APPROVED** [review]
Please fix the typos (or extend the checker), other than that LGTM

**Comment by setharnold** [comment]
This should address the spell-check errors, if I've done a good job of it anyway:

https://github.com/canonical/ubuntu-mir/pull/42

Thanks

**Review by didrocks — APPROVED** [review]
Once PR #42 to fix the spelling mistakes merged into that one, looks good to me!

**Comment by setharnold** [comment]
Ha, apparently I only addressed warnings, and not errors. So, here's one to fix `Mattermost`: https://github.com/canonical/ubuntu-mir/pull/43

**Comment by cpaelzer** [comment]
All spelling issues fixed, plenty of approvals and pre-discussed at the sprint. Merging ...

**Pull request merged**
